credentials/src/test/kotlin/SSITest.kt

```diff
@@ -146,17 +190,209 @@ class SSITest {
 
     val vcJwt: VcJwt = VerifiableCredential.create(signOptions, null, vc)
 
-    val createVpOptions = CreateVpOptions(getPresentationDefinition(), arrayListOf(vcJwt), did)
+    val btcAddressPd = buildPresentationDefinition(
+      inputDescriptors = listOf(
+        buildInputDescriptor(fields = listOf(buildField(paths = arrayOf("$.credentialSubject.btcAddress"))))
+      )
+    )
+
+    val createVpOptions =
+      CreateVpOptions(btcAddressPd, arrayListOf(vcJwt), SimpleResolver(didDocument), did)
 
     val exception = assertThrows<Exception> {
       VerifiablePresentation.create(signOptions, createVpOptions)
     }
 
     assertTrue(
       exception
-        .message?.contains("There are no useable Vcs that correspond to the presentation definition")!!
+        .message?.contains("is not satisfied in InputDescriptor")!!
```

Comment:
Seems to me like this is what you'd want?
```suggestion
        .message!!.contains("is not satisfied in InputDescriptor")
```